cwd/views/index.pug

```diff
@@ -37,4 +37,5 @@ html
     script const activeGlyphs = !{activeGlyphsStr};
     script const API_URL = "!{api_url}";
     script(src="./dist/cwd-bundle.js")
+    script(src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js")
```

Comment:
One thought now that it's working is do you need Ajax, or could you just use the fetch API like I use elsewhere?

Reply:
I think I can, not sure of the syntax but I can figure it out. Would you prefer that?

Reply:
Yeah, I like for either all requests to be AJAX or all requests to be fetch api.  I like the fetch api better honestly.

Reply:
Sounds good! I'll clean this all up.